components/ledger/internal/adapters/database/postgres/account.postgresql.go

```diff
@@ -229,14 +229,14 @@ func (r *AccountPostgreSQLRepository) Find(ctx context.Context, organizationID,
 }
 
 // FindByAlias find account from the database using Organization and Ledger id and Alias.
```

Comment:
```suggestion
// FindByAlias find account from the database using Organization and Ledger id and Alias. Returns true and ErrAliasUnavailability error if the alias is already taken.
```

Reply:
May be better to understand the func return as the func name doesn't explicit says it's looking for duplicated registers and if true returns an error.